pkg/ui/query.go

```diff
@@ -3,6 +3,8 @@ package ui
 import (
 	"html/template"
 	"net/http"
+	"path"
+
```

Comment:
can we invoke `make format`? It would be nice (and exclude the changes generated by it and not related to the PR)

Reply:
I use Go extension for Visual Studio Code already. It formats code automatically.
I ran `make format`, but it didn't change any of the files in this PR.